_posts/en/newsletters/2022-04-27-newsletter.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Bitcoin Optech Newsletter #197'
+permalink: /en/newsletters/2022/04/27/
+name: 2022-04-27-newsletter
+slug: 2022-04-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Discussion about activating CTV:** Jeremy Rubin [posted][rubin
+  ctv-st] to the Bitcoin-Dev mailing list his plan to release software
+  that will allow miners to begin signaling whether they intend to
+  enforce the [BIP119][] rules for the proposed
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV) opcode.
+  If 90% of blocks signal positively during any of the next several
+  2,016-block (two-week) retarget periods, anyone using Rubin's software
+  will themselves begin enforcing CTV's rules starting around early
+  November.
+
+  Rubin described in detail multiple reasons for believing Bitcoin users
+  may wish to activate CTV now, such as:
+
+  - *Consistency:* CTV has a stable specification and implementation
+
+  - *Popularity:* a number of people and organizations who are well
+    known to the Bitcoin community support CTV
+
+  - *Viability:* there do not appear to be any objections claiming that
+    CTV violates any highly desired properties of Bitcoin
+
+  - *Desirability:* CTV provides features users want, such as
+    [covenant][topic covenants]-based [vaults][topic vaults]
+
+  Over a dozen people replied on the mailing list, either directly to
+  Rubin's email or in other threads.  We're unable to summarize all the
+  noteworthy replies, but some of the comments we found particularly
+  interesting included:
+
+  - Anthony Towns [analyzed][towns ctv signet] transactions on the CTV
+    [signet][topic signet].  Almost all looked like they may have been
+    constructed with the same software ([Sapio][]), possibly indicating
+    lack of public exploration of CTV.  He further noted that changing
```

Comment:
FWIW, there has since been activity on signet using a modified version of the simple-ctv-vault software: https://twitter.com/fiatjaf/status/1517836181240782850